Describe env for runnable JARs (#6328)
OpenLiberty/docs: modules/ROOT/pages/runnable-jar-files.adoc

You can specify multiple variables with this format.
Variables that are specified on the command line override all other sources of variables and can't be changed after the server starts.

== JVM processes and environment

> By default, the Open Liberty server runs in the JVM that created immediately from the command line `java -jar ...` command.  In this case, JVM options are read from the locations that are specific to this JVM: the command-line and any JVM-defined locations.  In this single-JVM case, environment variables are read from the current shell environment, and Liberty-specific configuration files like `server.env` are not relevant.

Reviewer: ```suggestion
By default, the Open Liberty server runs in the JVM that is created by running the `java -jar ...` command.  In this case, JVM options are read from the locations that are specific to this JVM: the command-line and any JVM-defined locations.  In this single-JVM case, environment variables are read from the current shell environment, and Liberty-specific configuration files like `server.env` are not relevant.
```

> By default the Open Liberty server will simply run in the JVM created immediately from the command line `java -jar ...` command.  In this case JVM options will be read from the locations specific to this JVM (the command-line and any JVM-defined locations).  In this single JVM case, environment variables will be read from the current shell environment, and Liberty-specific configuration files like `server.env` are not relevant.

Reviewer: ```suggestion
By default, the Open Liberty server runs in the JVM that created immediately from the command line `java -jar ...` command.  In this case, JVM options are read from the locations that are specific to this JVM: the command-line and any JVM-defined locations.  In this single-JVM case, environment variables are read from the current shell environment, and Liberty-specific configuration files like `server.env` are not relevant.
```

> However, if a `jvm.options` file is detected in any of the xref:reference:config/server-configuration-overview.adoc[locations] Liberty searches then a second JVM will be launched to run the Liberty server, using these options.   In this case a server.env may be used to provide the environment for the second, server JVM.

Reviewer: ```suggestion
However, if a `jvm.options` file is detected in any of the xref:reference:config/server-configuration-overview.adoc[locations] that Liberty searches, then a second JVM is launched to run the Liberty server with the specified  options. In this case, a `server.env` file can provide the environment for the second server JVM.
```

> A second JVM will likewise be used if the `WLP_JAR_DEBUG` environment variable is set, as described below.

Reviewer: ```suggestion
A second JVM is likewise used if the `WLP_JAR_DEBUG` environment variable is set, as described in the following section.
```
